specification/maps/data-plane/DataRegistry/preview/2022-12-01-preview/dataregistry.json

```diff
@@ -140,7 +140,7 @@
     "/dataRegistries/{udid}": {
       "get": {
         "description": "**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\nThe `Get Data Registry` request returns the data registry information, as an [AzureBlob](#azureblob) object in JSON format, in the body of the response along with the HTTP status code of 200 when successful. If Azure Maps cannot access the underlying content or the integrity check fails, the `status` is set to `Failed` and and error is returned.\n\nThe data registry service endpoint is limited in scope to the region of your Azure Maps account and is not available for global Azure Maps accounts.\n\n## Submit Get Data Registry Request\n\nA data registry `GET` request returns details of the requested data registry, as determined by the specified `udid`. The details are returned in the body of the response in JSON format.",
-        "operationId": "DataRegistries_GetDataRegistry",
+        "operationId": "DataRegistry_GetDataRegistry",
```

Comment:
It seems this needs to be changed to `DataRegistery_Get` to fix the LintDiff error that prohibits noun phrases after the underscore. cc/ @mikekistler

Reply:
I don't think "GetDataRegistry" is a noun phrase, but it has been many years since I had any formal English training so I could be wrong. I would have said that "GetDataRegistry" was a verb phrase -- a verb with a direct object.

Reply:
Well, the direct object is a noun, but the point is the Swagger LintDiff rule that is failing (and blocking me from merging) is looking for Noun_Verb only, not Noun_VerbDirectObject
https://github.com/Azure/azure-rest-api-specs/pull/22583/checks?check_run_id=11305548696

Since all other operation IDs here follow that pattern, this must be the culprit.

If we want to keep this as is that's fine but we'll need someone to override the failing check and merge.

Reply:
In the same swagger, there is an operationId, DataRegistry_GetContent, which passes the check.
So there may be something wrong to the LintDiff check.

@tjprescott
Can you help to find someone to override this check and merge this PR?

Reply:
@tjprescott @mikekistler 
Per travis, we need someone who has the permission to bypass the fail check and merge.
Do you know who has that permission?

Thanks

Reply:
I do not have merge permissions.

Reply:
@chcmsft please follow the link described in the above chat:
![image](https://user-images.githubusercontent.com/5723682/220405657-b3888425-2b0b-4a88-b769-535ed6bc53e0.png)

We can't just force merge this otherwise every PR that is made henceforth will require a force merge.

Reply:
@tjprescott
thanks for the help. I've asked in the channel.

But I think we can still force merge this PR first. I did some tests and found that other changes to this swagger won't introduce the lintDiff fail. Thus, the concern of a force merge of following PR is resolved.

Can you help to force merge this time?

Thanks

Reply:
The swagger support folks should be able to do that. As I mentioned, I do not have permissions to force merge. I still think the simplest solution is to fix the operation id.

Reply:
@tjprescott 
The swagger support folk had bypassed the linkdiff fail. Please help to merge. Thank you!